tf-modules/archive/sf_event_sqs_to_db_records.tf

```diff
@@ -168,7 +168,7 @@ resource "aws_lambda_function" "sf_event_sqs_to_db_records" {
   handler          = "index.handler"
   runtime          = "nodejs12.x"
   timeout          = local.sf_event_sqs_lambda_timeout
-  memory_size      = 256
+  memory_size      = 512
```

Comment:
For posterity -- this  change was already made on the feature branch, updating here as we were occasionally seeing timeouts on this lambda in the test work.